neqo-http3/src/lib.rs

```diff
@@ -39,6 +39,28 @@ pub use transaction_server::TransactionServer;
 
 type Res<T> = Result<T, Error>;
 
+#[derive(Debug, PartialEq, PartialOrd, Ord, Eq, Clone, Copy)]
+pub struct QpackSettings {
+    max_table_size_decoder: u64,
+    max_table_size_encoder: u64,
+    max_blocked_streams: u16,
```

Comment:
This is one cases where it might be OK to have public fields.  It makes instantiating settings easier.  If you do that, maybe provide default values (`#[derive(Default)]` isn't entirely crazy here, even if that isn't necessarily optimal) so that people can do `QpackSettings { max_blocked_streams: 100, ...QpackSettings::default() }` or similar.  The advantage of public fields is that you get documentation.  The above is better than `QpackSettings::new(0, 0, 100)`, because you don't have to look up which argument goes where.